schemas/context/profile-directmarketing.schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/profile-directmarketing",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Profile Direct Marketing",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "description": "Direct Marketing addressing for profiles.",
+  "definitions": {
+    "profile-directmarketing": {
+      "properties": {
+        "xdm:address": {
```

Comment:
Keeping the field name as 'address' may be a little ambiguous for the marketer. If I am not wrong, this field would be accessible to the marketer through the path /address/*.
Can we have something on the lines of directMarketingAddress/targetedAddress? What do you think?

Reply:
Good catch, there is an issue here, let me fix it.